src/CMakeLists.txt

```diff
@@ -403,6 +403,15 @@ if(WITH_DPDK)
   list(APPEND ceph_common_deps common_async_dpdk)
 endif()
 
+if(WITH_PMDK)
```

Comment:
Replace WITH_PMEM with WITH_BLUESTORE_PMEM. Later it will be if (WITH_BLUESTORE_PMEM or WITH_RBD_CACHE_PMEM)